scality_manila_utils/export.py

```diff
@@ -0,0 +1,219 @@
+# Copyright (c) 2015 Scality
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import re
+
+from scality_manila_utils.exceptions import (ExportException,
+                                             DeserializationException)
+
+
+class ExportTable(object):
+    """
+    A set of exports that can be destilled into /etc/exports.
+    """
+    def __init__(self, exports):
+        """
+        Create a new exports table from the given exports.
+
+        :param exports: a list, tuple or other iterable of exports
+        :type exports: sequence of
+            :py:class:`scality_manila_utils.export.Export`
+        """
+        self.exports = dict(
+            (export.export_point, export)
+            for export in exports
+        )
+
+    def add_client(self, export_point, host, options=None):
```

Comment:
This is the terminology used in the man page. I could change `host` to `machine_name` to make it closer to the man page terminology, but I don't think that's much better.